cmd/kube-apiserver/app/server.go

```diff
@@ -393,6 +410,14 @@ func (s *APIServer) Run(_ []string) error {
 		}
 	}
 
+	databaseStorage := master.DatabaseClients{
+		Default:      etcdStorage,
+		Experimental: expEtcdStorage,
+	}
+	if etcdEventStorage != nil {
+		databaseStorage.Overwrites = map[string]storage.Interface{"events": etcdEventStorage}
```

Comment:
Suggest always creating the map, so that this becomes `databaseStorage.Overwrites["events"] = etcdEventStorage`